wiki/Client/Beatmap_editor/AiMod/en.md

```diff
@@ -25,139 +25,143 @@ Development of AiMod usually lags behind the standards held by the mapping commu
 
 #### Meta
 
-| Message | Explanation | Solution |
-| :-- | :-- | :-- |
-| HP rate for Easy/Normal is suggested to be at least 4. | osu!mania maps only! | Go to Song Settings and set HP rate to 4 or higher. |
-| HP rate for Hard and up is suggested to be at least 7. | osu!mania maps only! | Go to Song Settings and set HP rate to 7 or higher. |
-| OD rate is suggested to be at least 5. | osu!mania maps only! | Set `Overall Difficulty` in the "Difficulty" tab in Song Setup to 5 or higher. |
-| OD rate for maps with few sliders is suggested to be at least 7. | osu!mania maps only! | Go to Song Settings and set OD rate to 7 or higher. |
-| <!-- Both seem to occur, not sure what changes the value -->OD rate for maps with very few sliders is suggested to be at least 8. | osu!mania maps only! |  |
-| The Slider Velocity should be 1.40 or 1.60. | osu!taiko maps only! | Go to Timing tab and change Slider Velocity to 1.40 or 1.60. |
+| Message | Explanation | Solution | Notes |
+| :-- | :-- | :-- |  :-- |
+| HP rate for Easy/Normal is suggested to be at least 4. | *Self-explanatory* | Set the `HP Drain Rate` in the Difficulty tab of [Song Setup](/wiki/Client/Beatmap_editor/Song_Setup) to 4 or higher. | ![](/wiki/shared/mode/mania.png) osu!mania maps only |
+| HP rate for Hard and up is suggested to be at least 7. | *Self-explanatory* | Set the `HP Drain Rate` in the Difficulty tab of [Song Setup](/wiki/Client/Beatmap_editor/Song_Setup) to 7 or higher. | ![](/wiki/shared/mode/mania.png) osu!mania maps only |
+| OD rate is suggested to be at least 5. | *Self-explanatory* | Set the `Overall Difficulty` in the Difficulty tab of [Song Setup](/wiki/Client/Beatmap_editor/Song_Setup) to 5 or higher. | ![](/wiki/shared/mode/mania.png) osu!mania maps only. This warning can safely be ignored if the usage of the low Overall Difficulty in question is intended. |
+| OD rate for maps with very few sliders is suggested to be at least 7. | *Self-explanatory* | Set the `Overall Difficulty` in the Difficulty tab of [Song Setup](/wiki/Client/Beatmap_editor/Song_Setup) to 7 or higher. | ![](/wiki/shared/mode/mania.png) osu!mania maps only. This warning can safely be ignored if the usage of the low Overall Difficulty in question is intended. |
+| <!-- Both seem to occur, not sure what changes the value -->OD rate for maps with very few sliders is suggested to be at least 8. | *Self-explanatory* | Set the `Overall Difficulty` in the Difficulty tab of [Song Setup](/wiki/Client/Beatmap_editor/Song_Setup) to 8 or higher. | ![](/wiki/shared/mode/mania.png) osu!mania maps only. This warning can safely be ignored if the usage of the low Overall Difficulty in question is intended. |
+| The Slider Velocity should be 1.40 or 1.60. | *Self-explanatory* | Go to Timing tab and set the `Slider Velocity:` to either 1.40 or 1.60. | ![](/wiki/shared/mode/mania.png) osu!mania maps only |
```

Comment:
Should be "osu!taiko and osu!mania maps only"?

I tried to set SV in osu!taiko maps, and the editor shows the exact message.
![image](https://user-images.githubusercontent.com/110881926/231486412-1620e26b-d4ea-493f-b94c-7ffb97178069.png)

Reply:
It should be osu!taiko only not mania

Reply:
changed, thank you!